src/content/teammembers.yaml

```diff
@@ -0,0 +1,27 @@
+- id: Xanthous
```

Comment:
I think we should map Team members to Authors directly instead of creating another Yaml. it's sort of duplication of data which doesn't fit `DRY` concept.

Reply:
Agreed. This should just map to our `authors.yaml`